content/g-sgov/g-sgov-lidé/g-sgov-lidé-glosář.ttl

```diff
@@ -23,34 +23,29 @@ g-sgov-lidé:glosář a a-popis-dat-pojem:glosář, owl:Ontology, skos:ConceptSc
 g-sgov-lidé-pojem:člověk a skos:Concept;
   skos:broader <https://slovník.gov.cz/legislativní/sbírka/111/2009/pojem/subjekt-údajů-vedený-v-registru-obyvatel>,
     z-sgov-pojem:typ-objektu;
-  skos:definition "Fyzická osoba"@cs;
   skos:inScheme g-sgov-lidé:glosář;
-  skos:prefLabel "Člověk"@cs;
-  skos:scopeNote "Fyzická osoba (člověk) ve smyslu občanského zákoníku."@cs .
+  skos:prefLabel "Člověk"@cs .
 
 g-sgov-lidé-pojem:tituly-před-jménem a skos:Concept;
   skos:broader z-sgov-pojem:typ-vlastnosti, z-sgov-pojem:vlastnost;
-  skos:definition "Akademické či jiné tituly psané před jménem."@cs;
   skos:inScheme g-sgov-lidé:glosář;
-  skos:prefLabel "tituly před jménem"@cs .
+  skos:prefLabel "tituly před jménem"@cs;
+  skos:scopeNote "Určuje, zda je možné kouření tabákových výrobků."@cs .
 
 g-sgov-lidé-pojem:tituly-za-jménem a skos:Concept;
   skos:broader z-sgov-pojem:typ-vlastnosti, z-sgov-pojem:vlastnost;
-  skos:definition "Akademické či jiné tituly psané za jménem."@cs;
   skos:inScheme g-sgov-lidé:glosář;
-  skos:prefLabel "tituly za jménem"@cs .
+  skos:prefLabel "tituly za jménem"@cs;
+  skos:scopeNote "Určuje, zda je možné kouření tabákových výrobků."@cs .
 
 g-sgov-lidé-pojem:celé-jméno a skos:Concept;
   skos:broader z-sgov-pojem:typ-vlastnosti, z-sgov-pojem:vlastnost;
-  skos:definition "Celé jméno jako jeden řetězec."@cs;
-  skos:hiddenLabel "křestní jméno a příjmení"@cs;
   skos:inScheme g-sgov-lidé:glosář;
   skos:prefLabel "celé jméno"@cs;
-  skos:scopeNote "Může obsahovat křestní (osobní) jméno, příjmení,  další jména a tituly člověka."@cs .
+  skos:scopeNote "Určuje, zda je možné kouření tabákových výrobků."@cs .
```

Comment:
Toto neni korektni. @martinnec neopravoval si todle nahodou nekdy. Myslim ze jsem to posilal v slacku, no uz to neni v historii.